src/spicy/runtime-support.h

```diff
@@ -502,8 +502,8 @@ template<typename K, typename V>
 ValPtr to_val(const hilti::rt::Map<K, V>& s, const TypePtr& target);
 template<typename T>
 ValPtr to_val(const hilti::rt::Set<T>& s, const TypePtr& target);
-template<typename T>
-ValPtr to_val(const hilti::rt::Vector<T>& v, const TypePtr& target);
+template<typename T, typename Allocator>
+ValPtr to_val(const hilti::rt::Vector<Allocator>& v, const TypePtr& target);
```

Comment:
Should this not keep `T`? Or am I confused?

```suggestion
ValPtr to_val(const hilti::rt::Vector<T, Allocator>& v, const TypePtr& target);
```